sdcm/nemesis.py

```diff
@@ -2069,9 +2069,10 @@ def _prepare_test_table(self, ks='keyspace1', table=None):
         test_keyspaces = self.cluster.get_test_keyspaces()
         # if keyspace or table doesn't exist, create it by cassandra-stress
         if ks not in test_keyspaces or not table_exist:
+            ks_rf = min(self.tester.reliable_replication_factor, 3)
```

Comment:
why not apply this fix in `reliable_replication_factor` ?

Reply:
Done.